package.json

```diff
@@ -16,7 +16,7 @@
     "test": "npm run lint && npm run test-only",
     "test-only": "mocha -c --require test/setup.js --recursive ./test",
     "test-app": "./bin/nodejs-dashboard.js node test/app/index.js",
```

Comment:
@jasonwilson -- Can you comment here? The problem is that `bin/nodejs-dashboard.js` isn't a recognizable Windows cmd script.

I'm guessing we want:

```sh
node bin/nodejs-dashboard.js node test/app/index.js
```

but not totally sure...